### DISCUSSION TITLE
Getting the folder name

### DISCUSSION THREAD

**Discussion by colindavidfoster** [tfeldmann/organize#397] [Q&A] [answered]
How can I get the current folder from the path?

For example if {path} is \\\\microserver\\Downloads\\complete\\Test\\advert.mpg.zero
I get
\\\\microserver\\Downloads\\/complete
  Test/advert.mpg.zero
    - (echo) Path is \\\\microserver\\Downloads\\complete\\Test\\advert.mpg.zero
    - (echo) Relative Path is Test\advert.mpg.zero
    - (echo) Name is advert.mpg

How can I get \\\\microserver\\Downloads\\complete\\Test into, say, {folder} so that I can use something like
- shell: 'touch "{folder}\\{name}"'

Here's my rule.

```
rules:
  - name: Truncate To Zero
    locations:
      - path: \\\\microserver\\Downloads\\complete
    subfolders: true
    filters:
      - name:
      - extension: ["zero"]
    actions:
      - echo: Path is {path}
      - echo: Relative Path is {relative_path}
      - echo: Name is {name}
      #- shell: 'touch "{folder}\\{name}"'
      - delete:
    tags:
      - debug
```

**Comment by tfeldmann** [comment, marked as answer]
I think you‘re searching for `{path.parent}`?

**Reply by colindavidfoster** [reply]
Perfect, but not in the docs :-(

**Reply by tfeldmann** [reply]
We absolutely need better docs for this 👍
Right now there is only a link to pathlib.Path and the user is expected to know the rest.
